policy/predicate/file.go

```diff
@@ -59,6 +65,15 @@ func (pred *ChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (*co
 
 	changedFiles := []string{}
 
+	for _, f := range files {
+		if anyMatches(pred.RejectPaths, f.Filename) {
+			predicateResult.Values = []string{f.Filename}
+			predicateResult.Description = f.Filename + " was changed"
+			predicateResult.Satisfied = false
+			return &predicateResult, nil
+		}
+	}
```

Comment:
**question (non-blocking):** why do you have to do another loop here ? 🤔 You cannot do it in the initial `range` ?

Reply:
a this line the range is break with return https://github.com/palantir/policy-bot/pull/503/files#diff-9ab6ff99f800968214d5d7039c7db7b84b4ca6f859b85ee288fef0f37c8a7e79R89

R

Reply:
if a file is found before one of the rejected files, the rule will be validated when I don't want it

Reply:
Yes you're right 💯